File: LXDSlideMenu-master/LXDSlideMenuObjectiveC/LXDSlideMenuObjectiveC/LXDMainViewController.m
//
//  LXDMainViewController.m
//  LXDSlideMenuObjectiveC
//
//  Created by linxinda on 15/11/29.
//  Copyright © 2015年 sindriLin. All rights reserved.
//

#import "LXDMainViewController.h"
#import "LXDSlideManager.h"

@interface LXDMainViewController ()

@end

@implementation LXDMainViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    UIViewController * slideController = [[UIStoryboard storyboardWithName: @"Main" bundle: nil] instantiateViewControllerWithIdentifier: @"menuController"];
    LXDSlideManager * slideManager = [LXDSlideManager sharedManager];
    [slideManager setupMainController: self];
    [slideManager setupMenuController: slideController];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: LXDSlideMenu-master/LXDSlideMenuSwift/LXDSlideMenuSwift/LXDMainViewController.swift
//
//  LXDMainViewController.swift
//  LXDSlideMenuSwift
//
//  Created by linxinda on 15/11/29.
//  Copyright © 2015年 sindriLin. All rights reserved.
//

import UIKit

class LXDMainViewController: UITabBarController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let slideManager = LXDSlideManager.sharedManager()
        slideManager.setMainController(self)
        slideManager.setMenuController(UIStoryboard(name: "Main", bundle: nil).instantiateViewControllerWithIdentifier("menuController"))
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }

}
